tests/test_rules.py

```diff
@@ -681,6 +681,24 @@ def test_explicit_string_values_int():
     assert (String("0x123") in children) == True
 
 
+def test_string_values_special_characters():
+    rule = textwrap.dedent(
+        """
+        rule:
+            meta:
+                name: test rule
+            features:
+                - or:
+                    - string: hello\\r\\nworld
+                    - string: some\\path
```

Comment:
i'm glad that the yaml parser doesn't screw this up before it gets to us

Reply:
shouldn't these be quoted now? 😕